src/main/java/com/xiaomi/infra/pegasus/example/FullScanSample.java

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2020, Xiaomi, Inc.  All rights reserved.
+// This source code is licensed under the Apache License Version 2.0, which
+// can be found in the LICENSE file in the root directory of this source tree.
+package com.xiaomi.infra.pegasus.example;
+
+import com.xiaomi.infra.pegasus.client.*;
+import java.io.UnsupportedEncodingException;
+import java.util.List;
+import org.apache.commons.lang3.tuple.Pair;
+
+/** full scan sample */
+public class FullScanSample {
+
+  public static void main(String[] args) throws PException, UnsupportedEncodingException {
+    PegasusClientInterface client = PegasusClientFactory.createClient(ClientOptions.create());
+    PegasusTableInterface table = client.openTable("temp");
+
+    // Set up the scanners.
+    ScanOptions scanOptions = new ScanOptions();
+    scanOptions.batchSize = 20;
+    // Values can be optimized out during scanning to reduce the workload.
+    scanOptions.noValue = true;
```

Comment:
In former case, set `scanOptions.noValue = true` is because that case only need `hashKey/sortKey` to filter result. In this general case, this options may not be appropriate. And I see the follow code you add `table.get` to obtain the `value` which is redundant. I suggest you just explain the `scanOptions.noValue = true`, And use `pair.getRight` to get value in follow code.

Reply:
Ok， Got it！ I use `table.get `because `pari.getRight `has no value, I understand now, thanks~